src/test/java/test/integration/extension/AddDependencies.java

```diff
@@ -0,0 +1,20 @@
+package test.integration.extension;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines set of dependencies which will be added to deployed test archive.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AddDependencies {
+
+    /**
+     * List of maven artifact coordinates in the format of groupId:artifactId:version.
+     * @return
+     */
+    String[] value();
```

Comment:
Yes and it makes a simple class full of "icing" on top like in Spring boot :) 

As for carriage returns - not really sure how probable this is to encounter such an issue having `g:a:v` spanning multiple lines.